notebooks/visualization/MetPy SkewT.ipynb

```diff
@@ -0,0 +1,298 @@
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Access archived sounding data\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first step in plotting a skew-T is accessing the sounding data. We're going to download some observed sounding data from the University of Wyoming sounding archive from 00z on 15 April 2015 at Topeka, KS during a major tornado outbreak. First, we create a datetime object for MetPy to use to select the correct time and station from the Wyoming archive."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dt = datetime(2012, 4, 15, 0)\n",
+    "station = 'TOP'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll download the data from the Wyoming archive into a Pandas dataframe with units. You'll need an internet connection for this step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = WyomingUpperAir.request_data(dt, station)\n",
+    "data = pandas_dataframe_to_unit_arrays(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll split that Pandas dataframe up into individual arrays for each variable with units, and calculate wind speed from the wind components."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p = data['pressure']\n",
+    "T = data['temperature']\n",
+    "Td = data['dewpoint']\n",
+    "u = data['u_wind']\n",
+    "v = data['v_wind']\n",
+    "spd = wind_speed(u,v)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Plot the Skew-T\n",
+    "\n",
+    "Now we'll plot the data we downloaded on a skew-T diagram.\n",
+    "\n",
+    "The first two lines of code in this cell create a figure using Matplotlib and then modifies it into a skew-T plot\n",
+    "using MetPy's SkewT class.\n",
+    "\n",
+    "The next three lines plot temperature and dewpoint data as red and green lines on the Skew-T, as well as plotting the wind data as barbs along the right side of the plot.\n",
+    "\n",
+    "The two lines below that set the limits of the plot. These are set up to generally be good for most soundings,\n",
+    "but you may want to adjust them, especially if you are plotting particularly warm or cold profile.\n",
+    "\n",
+    "The next three lines add dry adiabats, moist adiabats, and mixing lines to the plot.\n",
+    "\n",
+    "Finally, the last three lines add a title to the plot and tell matplotlib to display it."
+   ]
+  },
+  {
```

Comment:
Same with this section.  It sounds like maybe 3 separate markdown-code cell pairs would make this a little bit clearer to the user.

Reply:
Sounds good! I just re-structured this section and added the link to the python-awips notebook.

Reply:
Okay, very close! It's looking good, I would just remove most the the "line descriptions" from the main section header now, since you're addressing them each in their own markdown cell.
![Screen Shot 2021-01-08 at 10 45 43 PM](https://user-images.githubusercontent.com/21114460/104084337-43c07d00-5203-11eb-83e2-9fb04cf9679b.png)

And then in section 3 you can break it into the 3 markdown-code cell pairs that are described in the section description (but then move that from the section description). (1a is the markdown cell, followed by 1b as the code cell, followed by 2a, 2b, 3a, 3b).
![Screen Shot 2021-01-08 at 10 50 13 PM](https://user-images.githubusercontent.com/21114460/104084398-e37e0b00-5203-11eb-8b2d-00796ac93db9.png)